<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-FOMJMEG3" failures="1" tests="1" name="NegativeTest" time="7.666" errors="0" timestamp="2023-04-07T16:36:05 IST">
  <testcase classname="com.herokuapp.theinternet.NegativeTest" name="IncorrectUsernameTest" time="7.658">
    <failure type="java.lang.AssertionError" message="Actual error message does not contains expected.
Actual:Your username is invalid!
×
Expected:The username is valid expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Actual error message does not contains expected.
Actual:Your username is invalid!
×
Expected:The username is valid expected [true] but found [false]
at com.herokuapp.theinternet.NegativeTest.IncorrectUsernameTest(NegativeTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- IncorrectUsernameTest -->
</testsuite> <!-- NegativeTest -->
